rita_exit/src/rita_loop/mod.rs

```diff
@@ -358,6 +358,7 @@ pub fn start_rita_exit_endpoints(workers: usize) {
                     .route("/client_debt", web::post().to(get_client_debt))
                     .route("/time", web::get().to(get_exit_timestamp_http))
                     .route("/exit_list", web::post().to(get_exit_list))
+                    .route("/exit_list", web::get().to(get_exit_list_v2))
```

Comment:
please use exit_list_v2 as the endpoint string not a post vs get endpoint.

Reply:
done